hoot-services/src/main/java/hoot/services/controllers/grail/PullApiCommand.java

```diff
@@ -163,4 +160,20 @@ public static InputStream getHttpResponseWithSSL(String url) throws Exception {
 
         return entity.getContent();
     }
+
+    /**
+     * String query that will retrieve the connected ways.
+     * @return String query that will retrieve the connected ways.
+     */
+    static String connectedWaysQuery() {
+        String newQuery =
+            "(" +
+                "way({{bbox}});>;" + // get all ways within bbox and the nodes for those ways
+            ");" +
+            "way(bn);" + // select parent ways for the nodes
+            "(._;>;);" + // get the nodes assocaited with those ways as well
+            "out meta;";
```

Comment:
@brianhatchl should i instead store this in a separate file like the other query?

Reply:
I think that's a good idea.